code/online_challenges/src/rosalind/algorithmic_heights/fibonacci.py

```diff
@@ -0,0 +1,21 @@
+# Find the nth Fibonacci number
+def fib(n):
+
+    fibDict = dict()
```

Comment:
why we need to use dict() here?
- you can find the number using recursive technique as you suggested but directly with no need to load dictionary and make a sub-function.
- maybe you need to think about how to handle a negative input. but note that [fibonacci sequence](https://en.wikipedia.org/wiki/Fibonacci_number) can be extended to include the negative index "negafibonacci".
- and i think in python you don't need to use new lines that much, it's readable by syntax.

and a reminder, please make two different PRs if you commit in two different dirs.